### PR TITLE
Echo an empty string if grep fails in subshell

### PR DESCRIPTION
**Pull request by rekmarks**
This PR ensures that a subshell whose final command is `grep` always exits with a `0` code, preventing the entire action from blowing up.

I had already made this change in one place, but forgotten this instance.

### CODE REVIEW

File: scripts/download-artifact.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ fi
 
 OLDEST_PR_BRANCH_MERGE_COMMIT=$(
   git rev-list "$PREFIXED_PULL_REQUEST_BASE_BRANCH"..HEAD --merges --reverse |
-  grep -o -m 1 '\w\+'
+  grep -o -m 1 '\w\+' || echo ''
```
Comment (Gudahtt) [inline scripts/download-artifact.sh:62]: Usually if I want a line to be allowed to fail, I'll use `|| true`. That or `|| :` seem to be the two popular idioms. I guess this works too though!

**Review by Gudahtt — APPROVED** [review]
LGTM!

File: scripts/download-artifact.sh (hunk 1 above)
Comment (rekmarks) [reply scripts/download-artifact.sh:62]: In this case, I want the resulting variable to be an empty string, true.

Comment (Gudahtt) [reply scripts/download-artifact.sh:62]: The result would be an empty string with either of these three options.

**Pull request merged**